src/site/content/en/blog/samesite-cookies-explained/index.md

```diff
@@ -268,68 +268,77 @@ on your site, you should also check with the provider that they are updating
 their services. You may need to update your dependencies or snippets to ensure
 that your site picks up the new behavior. {% endAside %}
 
-These changes are all backwards-compatible with other browsers. By making these
-changes to your cookies, you are making their intended use explicit rather than
-relying on the default behavior of the browser. Likewise, any clients that do
-not recognize `SameSite=None` as of yet should ignore it and carry on as if the
-attribute was not set.
+These changes are backwards-compatible with browsers that have correctly
+implemented earlier versions `SameSite` or do not support it at all. By making
+these changes to your cookies, you are making their intended use explicit rather
+than relying on the default behavior of the browser. Likewise, any clients that
+do not recognize `SameSite=None` as of yet should ignore it and carry on as if
+the attribute was not set.
 
 Additionally in Chrome, if you also enable the
 `cookies-without-same-site-must-be-secure` flag then you must also specify
 `SameSite=None` cookies as `Secure` or they will be rejected. Note, this flag
 won't have any effect unless you also have `same-site-by-default-cookies`
 enabled.
 
+{% Aside 'warning' %} Warning: At the time of writing, the network library on
```

Comment:
Remove the "Warning:" as it is part of aside formatting (duplicate now). Also, replace 'You can track' with just 'Track' to reduce size of aside.

Reply:
Oh, yes! That was a copy/paste error - thank you!